Beat-Board/beatBoard.py

```diff
@@ -0,0 +1,117 @@
+from tkinter import *
+from playsound import playsound
+from threading import Thread
+
+
+class padSound:
+    def __init__(self, soundLocation):
+        self.soundLocation = soundLocation
+
+    def given_sound(self):
+        playsound(self.soundLocation)
+
+    def play_sound(self,event):
+        sound = Thread(target=self.given_sound)
+        sound.start()
+
+
+# All the locations of the sounds
+kickLocation  = 'Sounds/Kick.wav'
+hiHatLocation = 'Sounds/hiHat.wav'
+snareLocation = 'Sounds/snare.wav'
+pad1Location  = 'Sounds/Pad1.wav'
+pad2Location  = 'Sounds/Pad2.wav'
+pad3Location  = 'Sounds/Pad3.wav'
+pad4Location  = 'Sounds/Pad4.wav'
+pad5Location  = 'Sounds/Pad5.wav'
+pad6Location  = 'Sounds/Pad6.wav'
```

Comment:
fix the paths for these. One example: 

`./Beat-Board/Sounds/Kick.wav`

Reply:
I have also updated the paths as well